### PR TITLE
must-gather: collect OBC of all namespaces

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal nigoyal@redhat.com

cc @raghavendra-talur

### CODE REVIEW

**Comment by crombus** [comment]
/hold  because must-gather scope is only in openshift-storage.

@jarrpa can you suggest what to do here.

**Comment by raghavendra-talur** [comment]
IMO, the exception that was provided for collecting the PVC data in must-gather should also apply for OBC data collection, they belong to the same category.

File: must-gather/collection-scripts/gather_namespaced_resources [hunk 1]
```diff
@@ -108,3 +105,8 @@ done
 echo "collecting dump of oc get pvc all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
 { oc get pvc --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/pvc_all_namespaces"
 { oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect pvc --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
+
+# For obc of all namesspaces
```
Comment (raghavendra-talur) [inline must-gather/collection-scripts/gather_namespaced_resources:109]: s/namesspaces/namespaces

**Review by raghavendra-talur — CHANGES_REQUESTED** [review]

File: must-gather/collection-scripts/gather_namespaced_resources (hunk 1 above)
Comment (iamniting) [reply must-gather/collection-scripts/gather_namespaced_resources]: Done

**Review by raghavendra-talur — APPROVED** [review]

**Review by jarrpa — CHANGES_REQUESTED** [review]
I think this is fine, OBCs are analogous to PVCs. However, the second commit should be merged into the first. I also wouldn't mind if a separate commit was added to fix the PVC typo, either in this PR or a separate one.

/hold cancel

**Comment by iamniting** [comment]
> I think this is fine, OBCs are analogous to PVCs. However, the second commit should be merged into the first. I also wouldn't mind if a separate commit was added to fix the PVC typo, either in this PR or a separate one.
> 
> /hold cancel

Done in this one itsef.

**Review by crombus — APPROVED** [review]

**Review by jarrpa — APPROVED** [review]
/lgtm

**Pull request merged**